### PR TITLE
how to fix docker won't build backend services

### PR DESCRIPTION
**Pull request by iamdey**

### CODE REVIEW

File: doc/install/frontend-development.md [hunk 1]
```diff
@@ -157,3 +157,18 @@ or rebuild node-sass:
...
 ```
+
+**docker container can't build or start**
+
+the following error can appears when the host user (you) can use directly  docker and when a file is owned by root only:
+
+> Building api
+> ERROR: Couldn't connect to Docker daemon at http+docker://localunixsocket - is it running?
+
+It is mostly due to minio which updates its config file without asking.
```
Comment (MonsieurLanza) [inline doc/install/frontend-development.md]: Not sure about "mostly"

File: doc/install/frontend-development.md [hunk 2]
```diff
@@ -157,3 +157,18 @@ or rebuild node-sass:
 ```
 npm rebuild node-sass
 ```
+
+**docker container can't build or start**
+
+the following error can appears when the host user (you) can use directly  docker and when a file is owned by root only:
```
Comment (MonsieurLanza) [inline doc/install/frontend-development.md]: **T**he following… (missing cap)

File: doc/install/frontend-development.md [hunk 3]
```diff
@@ -157,3 +157,18 @@ or rebuild node-sass:
...
+
+> Building api
+> ERROR: Couldn't connect to Docker daemon at http+docker://localunixsocket - is it running?
+
+It is mostly due to minio which updates its config file without asking.
+
+You need to change the owner of the file (please don't commit) until we find a solution.
```
Comment (MonsieurLanza) [inline doc/install/frontend-development.md]: How ?

File: doc/install/frontend-development.md (hunk 2 above)
Comment (PabloPie) [reply doc/install/frontend-development.md]: "can appear" also, and the next phrase is not very explicit, I would say "when docker is not run as root"

File: doc/install/frontend-development.md (hunk 1 above)
Comment (PabloPie) [reply doc/install/frontend-development.md]: Not needed IMO.

**Comment by iamdey** [comment]
@gdchamal I think we need a new pass on the doc about docker and eventually minikube when stable

**Comment by PabloPie** [comment]
Sure, the thing is it maybe won't be needed as image building and publishing will be automated.

**Pull request merged**